msi/GitProduct.wxs

```diff
@@ -6,26 +6,27 @@
   <?define SixtyFourBit ?>
 <?endif?>
 
-<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'>
-  <Product Id='*' Name='Git' Language='1033' Manufacturer='The Git Development Community'
+<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'
+     xmlns:swid='http://wixtoolset.org/schemas/v4/wxs/tag'>
+  <Product Id='*' Name='Git for Windows' Language='1033' Manufacturer='The Git Development Community'
            Version='$(var.Version)'
            UpgradeCode='abbb3e1d-fea0-4b84-9585-b4dc13cbc1b4'>
-    <Package InstallScope='perMachine' Compressed='yes' />
+    <Package InstallScope='perMachine' Compressed='yes' Keywords='Git' />
+    <swid:Tag Regid='git-for-windows.github.io' />
 
     <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' />
     <MediaTemplate CabinetTemplate='git{0}.cab' CompressionLevel='high' EmbedCab='yes' />
 
+    <!-- Add/Remove Programs Properties -->
     <Property Id='ARPCONTACT' Value='https://github.com/git-for-windows/git/wiki/Contact' />
     <Property Id='ARPURLINFOABOUT' Value='https://git-for-windows.github.io/' />
     <Property Id='ARPNOMODIFY' Value='1' />
-
     <Property Id='ARPPRODUCTICON' Value='git.ico' />
-    <Icon Id='git.ico' SourceFile='git.ico' />
```

Comment:
It was moved. I like to keep the WiX file organized. Of course, `Product` is a required element and naturally, I place it right under `Wix`. But right after that I like to put the `Package`, `MajorUpgrade` and `Media` elements, followed by the `Icon` used in Add/Remove Programs a/k/a Programs and Features.

This is followed by all the `ARP*` properties (Add/Remove Programs) which show up in various columns in ARP / Programs and Features. Other MSI properties go next (e.g. `MSIFASTINSTALL`), followed by any provided WiX properties (such as the one referenced for detecting the required version of the .NET Framework for enabling Git Credentials Manager for Windows). Finally, I follow this with application installer-specific properties.

Now, this file is getting quite large. What _might_ be better is to split this file up so that the Git for Windows-specific properties are in a `GitProperties.wxs` file, for example. It might also be a good idea to remove all the UI stuff from `GitProduct.wxs` into a `GitUI.wxs` file as well.